src/core/thread/mesh_forwarder_mtd.cpp

```diff
@@ -35,15 +35,92 @@
 
 #include "mesh_forwarder.hpp"
 
+#include "common/locator-getters.hpp"
+
 namespace ot {
 
 otError MeshForwarder::SendMessage(Message &aMessage)
```

Comment:
Almost same function with the one in mesh_forwarder_ftd.cpp. Is it doable to put this function to mesh_forwarder.cpp to avoid duplication?